### PR TITLE
Add CoopScenario entity

### PR DESCRIPTION
**Pull request by Marc-Spector**

### CODE REVIEW

File: faf-commons-api/src/main/java/com/faforever/commons/api/dto/CoopType.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package com.faforever.commons.api.dto;
+
+public enum CoopType {
+
+  // Order is crucial
```
Comment (Brutus5000) [inline faf-commons-api/src/main/java/com/faforever/commons/api/dto/CoopType.java]: I don't think the order here is crucial, as we don't need to map it to an int anywhere

File: faf-commons-api/src/main/java/com/faforever/commons/api/dto/CoopFaction.java [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package com.faforever.commons.api.dto;
+
+public enum CoopFaction {
+
+  // Order is crucial
```
Comment (Brutus5000) [inline faf-commons-api/src/main/java/com/faforever/commons/api/dto/CoopFaction.java]: I don't think the order here is crucial, as we don't need to map it to an int anywhere

**Review by Brutus5000 — CHANGES_REQUESTED** [review]

**Review by Brutus5000 — APPROVED** [review]

**Comment by Brutus5000** [comment]
@Marc-Spector Is this still WIP?

**Comment by Marc-Spector** [comment]
No. The PR is ready to merge

**Review by Sheikah45 — APPROVED** [review]

**Pull request merged**